geniza/corpus/management/commands/googledoc_to_annotation.py

```diff
@@ -0,0 +1,389 @@
+"""
+Script to convert translation content from Google Docs template
```

Comment:
I think I'm missing context about the bigger picture of the setup for this script and how you imagine it will be run, what the workflow is. Do you expect the script to be run manually? Cron job? One-time or ongoing? etc.

I was initially going to ask for an explanation via github comments, but I think it would make sense to expand the docs and add that larger context here, and make sure it gets included in the sphinx docs.